### PR TITLE
[GEOT-6367] Url encoding issue for GetFeature operation having special chars in PropertyName

### PR DESCRIPTION
**Pull request by taba90**

### CODE REVIEW

**Comment by nmco** [comment]
@taba90 please don't remove the pull request default template checklist, instead you shoudl read it and check the options has needed.

**Comment by nmco** [comment]
Associated JIRA ticket:
- https://osgeo-org.atlassian.net/projects/GEOT/issues/GEOT-6367

**Comment by nmco** [comment]
Looks like this pull request contains unrelated commits:
image

**Comment by taba90** [comment]
Everything should be fine now

File: modules/unsupported/swing/src/test/java/org/geotools/swing/DefaultRenderingExecutorMultiTest.java [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
-
-        executor.cancel(id);
-        boolean gotFailed =
-                listener.await(WaitingRenderingExecutorListener.Type.FAILED, WAIT_TIMEOUT);
-        assertTrue(
-                gotFailed
-                        || listener.eventReceived(WaitingRenderingExecutorListener.Type.COMPLETED));
-    }
-}
+/*
```
Comment (aaime) [inline modules/unsupported/swing/src/test/java/org/geotools/swing/DefaultRenderingExecutorMultiTest.java]: Full file reformat? Seems newlines were changed.

Comment (taba90) [reply modules/unsupported/swing/src/test/java/org/geotools/swing/DefaultRenderingExecutorMultiTest.java]: fixed

**Comment by nmco** [comment]
Looks good thx @taba90 !

**Pull request merged**